test/bigtest/interactors/jobs/RunningJobsInteractor.js

```diff
@@ -22,3 +22,11 @@ export class RunningJobInteractor {
     return this.when(() => this.isPresent);
   }
 }
+
+@interactor
+export class RunningJobsInteractor {
```

Comment:
What about RunningJobsContainer or something like this one? Or maybe find a better name for RunningJobInteractor? It's really challenging to find difference between RunningJobInteractor and RunningJobsInteractor.

Reply:
Yeah, it can be challenging indeed. Will use `RunningJobsContainer` instead.

Reply:
Sounds good. Thanks